pages/common/mdp.md

```diff
@@ -0,0 +1,19 @@
+# mdp
+
+> A command-line, ncurses based tool to make presentations from markdown files.
+
+- Launch a presentation from a markdown file:
```

Comment:
I don't think "Launch" is much clearer than "Start". The main potential point of confusion here is where/how the presentation is shown. If this happens directly in the terminal window, I think it'd be nice to explicitly say that. Otherwise one might wonder if, e.g., a GUI window appears, or maybe it would open in a browser, or something like that.